### PR TITLE
add averaged positioning funcionality doc

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: documentation/how-to/gnss.en.md [hunk 1]
```diff
@@ -195,3 +195,22 @@ It is possible to provide a mock location via a separate android app to
 QField. There are several options for this, one of them is [Android NTRIP Client](https://play.google.com/store/apps/details?id=com.lefebure.ntripclient).
 
 To use this you have to [enable mock locations on your Android device](https://www.youtube.com/watch?v=v1eRHmMiRJQ).
+
+## Averaged positioning functionality
+
+Note: users must have locked the coordinate cursor to their position.
```
Comment (nirvn) [inline documentation/how-to/gnss.en.md:201]: We don't explain how to do that anywhere in the page 
```suggestion
Note: the coordinate cursor must be locked to the current location via the _lock to position_ button.
```

File: documentation/how-to/gnss.en.md [hunk 2]
```diff
@@ -195,3 +195,22 @@ It is possible to provide a mock location via a separate android app to
...
+
+There is a function that allows you to digitize using averaged positions.
+
+The survey will start by pressing the add vertex button,
```
Comment (nirvn) [inline documentation/how-to/gnss.en.md:205]:
```suggestion
The survey will start by pressing and holding the add vertex button,
```

File: documentation/how-to/gnss.en.md [hunk 3]
```diff
@@ -195,3 +195,22 @@ It is possible to provide a mock location via a separate android app to
...
+which will start collecting positions.
+
+While collecting, an indicator will appear on top of the cursor
```
Comment (nirvn) [inline documentation/how-to/gnss.en.md:208]:
```suggestion
While collecting, an indicator will appear on top of the coordinate cursor
```

File: documentation/how-to/gnss.en.md [hunk 4]
```diff
@@ -195,3 +195,22 @@ It is possible to provide a mock location via a separate android app to
...
+showing a text reflecting the current number of collected positions and, as well, a progress bar.
```
Comment (nirvn) [inline documentation/how-to/gnss.en.md:209]:
```suggestion
showing a text reflecting the current number of collected positions. If an averaged position minimum count requirement is active, a progress bar will also be present indicating the progress towards meeting that requirement.
```

File: documentation/how-to/gnss.en.md [hunk 5]
```diff
@@ -195,3 +195,22 @@ It is possible to provide a mock location via a separate android app to
...
+
+!![](../assets/images/positioning_averaged.webp)
+
+The progress bar reflects the advance towards meeting a user-defined minimum
```
Comment (nirvn) [inline documentation/how-to/gnss.en.md:213]:
```suggestion
To setting to activate an average position minimum count threshold can be found in QField settings's positioning panel.
```

File: documentation/how-to/gnss.en.md [hunk 6]
```diff
@@ -195,3 +195,22 @@ It is possible to provide a mock location via a separate android app to
...
+collected position count. This requirement can be toggled in the settings *positioning* tab:
```
Comment (nirvn) [inline documentation/how-to/gnss.en.md:214]:
```suggestion
When active, holding the add vertex button is not required, a short tap on the button will begin the collection of positions and automatically add the averaged position when the minimum count requirement is met.
```

**Comment by m-kuhn** [comment]
Cool, thanks
Can we have a summary of quality attributes (i.e. what does this do to `position_direction`, `position_horizontal_accuracy`, ... and their `gnss_` prefixed friends)

**Comment by lucienicolier** [comment]
@nirvn thanks for your review

**Comment by lucienicolier** [comment]
@m-kuhn Do you mean in this chapter or on variables chapter?

**Comment by m-kuhn** [comment]
in the variables chapter with a link from here?

**Comment by lucienicolier** [comment]
@m-kuhn  about the variables, is it possible to get the info of all the surveys? I mean when we request 3 surveys to make one measure have the information of them separately, or it will give only one?

**Comment by m-kuhn** [comment]
as far as I know it is only one but it is not yet written in the documentation *how* this one value is constructed (average, sum, some maybe not available, some maybe additionally available), @nirvn can shed more light here.

**Comment by nirvn** [comment]
With regards to _accuracy_, the {horizontal,vertical} values represents the mean of all collected points, which can be useful.

For direction, speed, etc. it's also a mean value, however that shouldn't really matter in this context as I can't see why someone would want to average out a points _while moving_ :)

**Comment by nirvn** [comment]
@m-kuhn , I think you've summed up the situation quite well sir :)

File: documentation/how-to/gnss.en.md [hunk 7]
```diff
@@ -214,4 +214,6 @@ If an averaged position minimum count requirement is active, a progress bar will
 To setting to activate an average position minimum count threshold can be found in QField settings's *positioning* panel.
 When active, holding the add vertex button is not required, a short tap on the button will begin the collection of positions and automatically add the averaged position when the minimum count requirement is met.
 
-!![](../assets/images/positioning_averaged_set.jpg)
\ No newline at end of file
+!![](../assets/images/positioning_averaged_set.jpg)
+
+In this case, when using [`@gnss_*` variables](../gnss/#additional-variables) it will give the mean of all collected points.
```
Comment (m-kuhn) [inline documentation/how-to/gnss.en.md:219]: @nirvn can you review this sentence (one thing I cannot answer is gnss or position variables)

Comment (nirvn) [reply documentation/how-to/gnss.en.md:219]: That's an accurate statement

Comment (m-kuhn) [reply documentation/how-to/gnss.en.md:219]: Just to be crystal clear to our doc readers, that means `gnss_` variabls are averaged, `position_` variables are not averaged?

Comment (nirvn) [reply documentation/how-to/gnss.en.md:219]: Na, both would be averaged.

Comment (m-kuhn) [reply documentation/how-to/gnss.en.md:219]: @lucienicolier can you add that?

File: documentation/how-to/gnss.en.md [hunk 8]
```diff
@@ -195,3 +195,25 @@ It is possible to provide a mock location via a separate android app to
...
+If an averaged position minimum count requirement is active, a progress bar will also be present indicating the progress towards meeting that requirement.
+
+!![](../assets/images/positioning-averaged.webp)
+
+To setting to activate an average position minimum count threshold can be found in QField settings's *positioning* panel.
+When active, holding the add vertex button is not required, a short tap on the button will begin the collection of positions and automatically add the averaged position when the minimum count requirement is met.
+
+!![](../assets/images/positioning_averaged_set.jpg)
+
+In this case, when using [`@gnss_*` variables or `@position_`](../gnss/#additional-variables) it will give the mean of all collected points.
```
Comment (m-kuhn) [inline documentation/how-to/gnss.en.md:219]:
```suggestion
When using [`@gnss_*` or `@position_` variables](../gnss/#additional-variables) on averaged positions, the variable will also represent the average over all collected samples.
```

**Review by m-kuhn — APPROVED** [review]
Slightly improved wording proposed

**Comment by lucienicolier** [comment]
@nirvn is it okay for you?

**Pull request merged**